app/models/certificate.rb

```diff
@@ -4,6 +4,19 @@ class Certificate < Aggregate
   validates_presence_of :usage, :value, :component
   belongs_to :component, polymorphic: true
 
+  scope :all_pollable, ->(environment) {
+    select(:id, :value, :usage, :created_at, :updated_at, :component_id, :enabled, :component_type, :in_use_at, :notification_sent)
+    .joins(sanitize_sql_for_assignment(['
+      LEFT JOIN msa_components
+       ON msa_components.id = certificates.component_id AND msa_components.environment = ?
+      LEFT JOIN sp_components
+       ON sp_components.id = certificates.component_id AND sp_components.environment = ?
+      INNER JOIN services
+       ON sp_components.id = services.sp_component_id OR msa_components.id = services.msa_component_id
+      ', environment, environment]))
+    .where(in_use_at: nil, enabled: true)
```

Comment:
`enabled: true` is always true on encryption certs (i.e. they never get disabled). So this will keep returning disabled encryption certs...
btw, is the raw SQL neccessary? couldn't we make use of ActiveRecord?

Reply:
Thanks for your observation on disabled encryption certificates, I am open to suggestions on how we can do this efficiently with ActiveRecord?